WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -288,18 +288,24 @@ class LoginPrologueViewController: LoginViewController {
                        createSiteButton]
         } else if configuration.enableWPComLoginOnlyInPrologue {
             buttons = [continueWithWPButton]
+        } else if configuration.enableSiteAddressLoginOnlyInPrologue && configuration.enableSiteCreation {
+            buttons = [enterYourSiteAddressButton, createSiteButton]
+        } else if configuration.enableSiteAddressLoginOnlyInPrologue {
+            buttons = [enterYourSiteAddressButton]
         } else if configuration.enableSiteCreation {
             let createSiteButtonForBottomStackView = StackedButton(using: createSiteButton,
                                                                    stackView: .bottom)
             buttons = [continueWithWPButton,
                        enterYourSiteAddressButton,
                        createSiteButtonForBottomStackView]
         } else {
-            WPAuthenticatorLogError("Failed to create `StackedButton`s in login progue screen.")
+            WPAuthenticatorLogError("Failed to create `StackedButton`s in login prologue screen.")
             buttons = []
         }
 
-        let showDivider = configuration.enableWPComLoginOnlyInPrologue == false && configuration.enableSiteCreation == true
+        let showDivider = configuration.enableWPComLoginOnlyInPrologue == false &&
```

Comment:
Not related to this PR:

In a separate PR, I think we should remove this `enableWPComLoginOnlyInPrologue` flag as we no longer use that in Woo.

Reply:
@selanthiraiyan  We also added a few other configs for the simplified login project that are no longer used: `enableSocialLogin`, `emphasizeEmailForWPComPassword`, `wpcomPasswordInstructions` - do you think we should remove them too? That would require reverting the logic changes too.

Reply:
I think yes. IMO, cleaning up the unused logic will help to maintain the project and make future changes.

Reply:
Is there a safe way to test the removal to ensure it doesn't affect other host apps? We can't be sure that we are the only app that uses these configs.